src/org/omegat/core/team2/impl/GITRemoteRepository2.java

```diff
@@ -201,15 +199,15 @@ protected String getCurrentVersion() throws IOException {
     public void switchToVersion(String version) throws Exception {
         try (Git git = new Git(repository)) {
             if (version == null) {
-                version = REMOTE_BRANCH;
+                version = REMOTE + "/" + getDefaultBranchName(repositoryURL);
                 // TODO fetch
                 git.fetch().setRemote(REMOTE).setTimeout(TIMEOUT).call();
             }
             Log.logDebug(LOGGER, "GIT switchToVersion {0} ", version);
             git.reset().setMode(ResetType.HARD).call();
             git.checkout().setName(version).call();
-            git.branchDelete().setForce(true).setBranchNames(LOCAL_BRANCH).call();
-            git.checkout().setCreateBranch(true).setName(LOCAL_BRANCH).setStartPoint(version).call();
+            git.branchDelete().setForce(true).setBranchNames(getDefaultBranchName(repositoryURL)).call();
+            git.checkout().setCreateBranch(true).setName(getDefaultBranchName(repositoryURL)).setStartPoint(version).call();
```

Comment:
Instead of calling `getDefaultBranchName` repeatedly, can we call it once early on and keep using the same value? I think it would be easier to read that way.